launch/launch/actions/execute_process.py

```diff
@@ -308,27 +308,35 @@ def printer(context, msg, timeout_substitutions):
             ('float(', *self.__sigterm_timeout, ') + float(', *self.__sigkill_timeout, ')')
         )]
         # Setup a timer to send us a SIGTERM if we don't shutdown quickly.
-        self.__sigterm_timer = TimerAction(period=sigterm_timeout, actions=[
-            OpaqueFunction(
-                function=printer,
-                args=(base_msg.format('{}', '{}', 'SIGINT', 'SIGTERM'), sigterm_timeout)
-            ),
-            EmitEvent(event=SignalProcess(
-                signal_number=signal.SIGTERM,
-                process_matcher=matches_action(self)
-            )),
-        ])
+        self.__sigterm_timer = TimerAction(
```

Comment:
I had to reformat this because of line length after adding cancel_on_shutdown so the diff is big :-( sorry